src/devices/GrovePi/Models/PinMode.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace Iot.Device.GrovePiDevice.Models
+{
+    /// <summary>
+    /// Defines the pin mode, either input, either output
+    /// </summary>
+    public enum PinMode
+    {
+        Input = 0,
+        Output = 1
+    }
+
+    /// <summary>
+    /// Defines the pin level, either high, either low
+    /// </summary>
+    public enum PinLevel
```

Comment:
why not use PinValue?

Reply:
I meant that we already have it in Gpio namespace

Reply:
yes, I got it. I used the Gpio ones.